lms/static/notes.js

```diff
@@ -0,0 +1,76 @@
+function getPrivacyLevel(inputRange) {
+  const options = window.notesOptions.split(',');
+  const privacyText = document.getElementById('privacy-text');
+  privacyText.innerText = options[inputRange.value];
+  inputRange.addEventListener('change', () => {
+    privacyText.innerText = options[inputRange.value];
+  });
+}
+
+function trackAddNote(formElement) {
+  formElement.addEventListener('submit', () => {
+    const serializeData = $('#notes-form').serialize();
```

Comment:
Adding jQuery just for serialization is an enormous overhead. Frontend developers usually try to eliminate every additional kb sent over the internet. Try to avoid it.